index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <h1 id="header">
+      <meta charset="UTF-8">
+      <title>Pair Programming Two</title>
+      <link rel="stylesheet" href="css/style.css">
+      <nav id="nav-bar" >
+        <input href="#header" class = "nav-link" type="button" onclick="checkName()" value="Check A" />
+        <input href="#header" class = "nav-link" type="button" onclick="checkName()" value="Check B" />
+        <input href="#header" class = "nav-link" type="button" onclick="checkName()" value="Check C" />
+      </nav>
+
+      <img src="https://cdn.logo.com/hotlink-ok/logo-social.png" id="header-img">
+
+      <video src="https://www.youtube.com/watch?v=qz0aGYrrlhU" id="video"></video>
+    </h1>
+
+    <style type="text/css">
+            @media only screen and (min-width: 480px) and (max-width: 680px) {}
+    </style>
+
+  </head>
+
+  <body>
+
+      <form action="https://www.freecodecamp.org/email-submit" id ="form">
+        <label for="text">Please fill your email</label>
+
+        <input name="email" type="email" id ="email" placeholder="write something">
+        <input type="submit" value="submit" id ="submit">
+      </form>
+
+      <br><br>
+
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    Hello World, we are software developers!
```

Comment:
Please, make sure your code is clean when sent for review. This type of comment makes it hard for the reviewer (or other viewers) to place his/her attention on the important part of your code.